chia/wallet/wallet_node.py

```diff
@@ -466,41 +466,40 @@ async def _messages_to_resend(self) -> List[Tuple[Message, Set[bytes32]]]:
     async def _retry_failed_states(self):
         while not self._shut_down:
             try:
-                await asyncio.sleep(5)
-                current_time = time.time()
-                if self.last_state_retry_time < current_time - 10:
```

Comment:
can `self.last_state_retry_time` change from another task/coroutine? If so, it might not be safe to assume that sleeping 10 seconds is the same as this loop, checking the retry time regularly.

Reply:
No it can't this PR removes all occurrences of it.